spaces/sogur/index.json

```
@@ -0,0 +1,27 @@
+{
+  "key": "sogur",
+  "name": "Sogur",
+  "chainId": 1,
+  "decimals": 18,
+  "symbol": "SGR",
```

Adding indication that SGN token holders can also vote, following @e-asia's comment
```suggestion
  "symbol": "SGR+SGN",
```

---

Could you commit the change, i cant't applied the suggestion. Thank you.

---

done